articles/cosmos-db/set-throughput.md

```diff
@@ -56,7 +56,7 @@ All containers created inside a database with provisioned throughput must be cre
 
 If the workload on a logical partition consumes more than the throughput that's allocated to a specific logical partition, your operations are rate-limited. When rate-limiting occurs, you can either increase the throughput for the entire database or retry the operations. For more information on partitioning, see [Logical partitions](partition-data.md).
 
-Multiple logical partitions that belong to different containers that share the throughput provisioned to a database can be hosted on a single physical partition. While a single logical partition of a container is always scoped within a physical partition, *"L"* logical partitions across *"C"* containers that share the provisioned throughput of a database can be mapped and hosted on *"R"* physical partitions. 
+The shards of shared database containers are co-located, and up to 25 containers can potentially use the provisioned throughput. Beyond 25 containers, for every additional 25 containers added throughput accessible for all the containers reduces linearly
```

Comment:
Isn't the term shard specific to MongoDB, should it be partition instead? 

question - thoroughput reduces for every new container or for every 25 new containers?

Reply:
(1) We do not use the terms "shards" anywhere in our docs, so let's not introduce a new term here.
(2) "Potentially use" --> "use" 
(3)  Beyond 25 containers, for every additional 25 containers added throughput accessible for all the containers reduces linearly --> Beyond that, for every additional 25 containers added to the shared throughput database, the throughput accessible for all the containers in that database reduces linearly.

I will be happy to look at another iteration of this PR. Thanks.